cmd/image/qcow2ova/ova/ova.go

```diff
@@ -33,15 +35,18 @@ type OVA struct {
 	ImageName, VolumeName string
 	SrcVolumeSize         int64
 	TargetDiskSize        int64
+	PvsadmVersion         string
 }
 
 // Render will generate the OVA spec from the template with all the required information like image name, volume name
 // and size
 func Render(imageName, volumeName string, srcVolumeSize int64, targetDiskSize int64) (string, error) {
 	//Disk Size should be in bytes
+	PvsadmVersion := version.Get()
 	o := OVA{
-		imageName, volumeName, srcVolumeSize, targetDiskSize * 1073741824,
+		imageName, volumeName, srcVolumeSize, targetDiskSize * 1073741824, PvsadmVersion,
```

Comment:
wondering if we can just do this?
```suggestion
		imageName, volumeName, srcVolumeSize, targetDiskSize * 1073741824, version.Get(),
```